v-next/hardhat-utils/src/ci.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Returns 'true' if the current process is running in the GitHub CI environment.
+ */
+export function isCI() {
```

Comment:
This can be either a const or a function, what do you prefer?

Note: if we go for a const, how should we handle the tests? 
Because the module "ci" should be dynamically imported for every test because first the ENV variables need to be modified. And we might have to use the ugly solution to force the reload of the module:

 
![image](https://github.com/NomicFoundation/hardhat/assets/18092467/9a829556-b3e1-4a41-b027-707464b46c70)

Reply:
let's go for a function